inc/class_core.php

```diff
@@ -519,6 +521,8 @@ public function get_asset_url($path = '', $use_cdn = true)
 			$url = $path;
 		}
 
+		$plugins->run_hooks('get_asset_url_end', $url);
```

Comment:
@Sama34 Some CDNs might use different URLs. The main ones (AWS S3, etc) don't, but some might. This hook is there just in case.